src/main/webapp/feed.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>Message Feed</title>
-<link rel="stylesheet" href="/css/main.css">
-<link rel="stylesheet" href="/css/user-page.css">
-<script src="/js/feed-loader.js"></script>
-
-</head>
-<body onload="buildUI()">
- <div id="content">
-  <h1>Message Feed</h1>
-  <hr/>
-  <div id="message-container">Loading...</div>
- </div>
-</body>
-</html>
\ No newline at end of file
+  <head>
+	<title>Message Feed</title>
+	<link rel="stylesheet" href="/css/main.css">
+	<link rel="stylesheet" href="/css/user-page.css">
+	<script src="/js/feed-loader.js"></script>
+  </head>
+  <body onload="buildUI()">
+ 	<div id="content">
+ 	  <h1>Message Feed</h1>
+          <hr/>
+  	<div id="message-container">Loading...</div>
+    </div>
```

Comment:
Ooh, actually, this should line up with the `<div id="content">` it's closing from line 10. Anyway, thanks for cleaning this up! 👍

Reply:
<img width="439" alt="screen shot 2019-03-02 at 4 25 12 pm" src="https://user-images.githubusercontent.com/37354543/53687838-c626ae80-3d07-11e9-956b-1a6b1378b80d.png">
Yeah I'm confused bc on my text editor everything lines up but then when I pushed it everything is messed up again? I don't know why sublime doesn't like HTML

Reply:
Hmm--maybe a tab vs spaces thing? It's tricky to see in GitHub, but from trying to select the whitespace that's one thing I'd check. Maybe try setting `translateTabsToSpaces` true? http://www.sublimetext.com/docs/indentation